pkg/reconciler/openshift/tektonresult/extension.go

```diff
@@ -45,12 +68,33 @@ func (oe openshiftExtension) Transformers(comp v1alpha1.TektonComponent) []mf.Tr
 		occommon.ApplyCABundles,
 	}
 }
+
 func (oe openshiftExtension) PreReconcile(ctx context.Context, tc v1alpha1.TektonComponent) error {
 	return nil
 }
-func (oe openshiftExtension) PostReconcile(context.Context, v1alpha1.TektonComponent) error {
-	return nil
+func (oe openshiftExtension) PostReconcile(ctx context.Context, tc v1alpha1.TektonComponent) error {
+	return oe.installerSetClient.PostSet(ctx, tc, oe.routeManifest, filterAndTransform())
```

Comment:
always pass copy of `oe.routeManifest`
do not pass the reference

Reply:
Why?

Reply:
the receiver may change the reference directly and you have to use the same reference on multiple reconcile loops

Reply:
Oh. So this function is changing the reference. I just noticed. Probably, the function itself shouldn't receive a pointer but a value.

Reply:
But for now, I think we can change this to value.